contracts/mocks/PausableCrowdsaleImpl.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.18;
+
+import "../token/ERC20/ERC20.sol";
+import "../crowdsale/PausableCrowdsale.sol";
+
+
+contract PausableCrowdsaleImpl is PausableCrowdsale {
+
+    constructor (
+        uint256 _rate,
+        address _wallet,
+        ERC20 _token
+    )
```

Comment:
Again, single line (please delete the empty lines, why the added vertical whitespace?)